### ISSUE TITLE
light formatting of candidates

### ISSUE THREAD

**Issue by jkitchin** [tmalsburg/helm-bibtex#393]
Is there a way to add faces to parts of the candidates, e.g. make the authors be italicized, or add colors to the journal name, etc. this could make it easier to see different parts of the candidates. 

update: I have used an ivy transformer to fontify optional org-syntax in bibtex-completion-display-formats. I guess you could also do something similar with html markup in these. This can be handled outside of bibtex-completion, but I still wonder if you have any ideas other than these.

**Comment by tmalsburg** [comment]
Good idea to use shr. Some DSL that could also be used elsewhere would be nice.

**Comment by bdarcus** [comment]
#375 is likely relevant.
